source/material_model/visco_plastic.cc

```diff
@@ -56,10 +56,18 @@ namespace aspect
        */
       bool plastic_yielding = false;
 
+      MaterialModel::MaterialModelInputs <dim> in_new (in_new.n_evaluation_points(), this->n_compositional_fields());
```

Comment:
Nice fix for the deprecated version of the function! I would still prefer to call it `in` not `in_new`. There is no other `in` here, and additions like `_new` or `_2` are almost always bad ideas, because they carry no meaning.

Reply:
Thanks, Rene for reviewing! I have modified the variable names in the latest commit.